hrabsence/CRM/HRAbsence/BAO/HRAbsenceType.php

```diff
@@ -169,10 +189,49 @@ public static function del($absenceTypeId) {
       CRM_Core_BAO_OptionValue::del($id);
     }
 
+    // Delete Absence Type
     $absenceType->delete();
 
     CRM_Utils_Hook::post('delete', 'HRAbsenceType', $absenceTypeId, $absenceType);
   }
+  
+  /**
+   * Count leave requests for given activity type id
+   * 
+   * @param int $activityTypeID
+   *   ID of activity type to be used to count
+   * @return int
+   *   Number of activities found with given activity type ID
+   */
+  private static function countLeaveRequests($activityTypeID) {
```

Comment:
Since the method counts based on a parameter, it's better to put that on the method name: `countLeaveRequestsByActivityType()`